data_steward/cdr_cleaner/cleaning_rules/deid/registered_cope_survey_suppression.py

```diff
@@ -63,8 +64,12 @@ def get_suppressed_concept_ids(self):
         """
         with open(RT_COPE_SUPPRESSION_CSV_PATH) as f:
             concept_ids_df = read_csv(f, delimiter=',')
-            concept_ids = concept_ids_df['concept_id'].to_list()
-        return concept_ids
+            rt_concept_ids = concept_ids_df['concept_id'].to_list()
+        # From DC-2111
+        with open(RT_CT_COPE_SUPPRESSION_CSV_PATH) as f:
+            concept_ids_df = read_csv(f, delimiter=',')
+            rt_ct_concept_ids = concept_ids_df['concept_id'].to_list()
+        return list(set(rt_concept_ids) | set(rt_ct_concept_ids))
```

Comment:
Love the use of sets!

Reply:
Haha, thanks. It was confusing initially because duplicates blow up the count. If I try to keep the lists mutually exclusive, figuring out which concept came from which ticket would be more difficult. So kept them intact with union.